src/libs/SuggestionUtils.js

```diff
@@ -26,4 +26,18 @@ function trimLeadingSpace(str) {
     return str.slice(0, 1) === ' ' ? str.slice(1) : str;
 }
 
-export {getMaxArrowIndex, trimLeadingSpace};
+/**
+ * Checks if space is available to render large suggestion menu
+ * @param {Number} listHeight
+ * @param {Number} composerHeight
+ * @returns {Boolean}
+ */
+function spaceAvailableForLargeSuggestionMenu(listHeight, composerHeight) {
```

Comment:
This util won't hurt! But I meant something else, I wasn't clear enough

Reply:
Also, the name sounds like this returned some calculated space; maybe it could be name `hasEnoughSpaceForLargeSuggestionMenu` or something